mkdocs.yml

```diff
@@ -59,9 +59,11 @@ watch:
 
 nav:
     - Home: index.md
+    - Awesome: awesome_list.md
```

Comment:
I'm not sure I prefer `Awesome` or something else like `External Resources`. Also, unsure if it should be the second item here, or the last item.

Reply:
External Resources would also work for me.
Moving it to the end would also work for me.

Please let me know what I should change if any. Thanks.